integration/config/ias

```diff
@@ -0,0 +1 @@
+../../config/ias
```

Comment:
can't we create this at build time instead? (I mean the folder)

Reply:
No, that's where you store your IAS credentials (or are you suggesting that each build registers anew with IAS and fetches some api keys ;-?)

Reply:
Actually, were you referring to create the symlink to /config/ias at build time? What would be the advantage?